test/index.test.ts

```diff
@@ -4,12 +4,14 @@ describe('disposableEmailBlocklist', () => {
   test('should return an array of strings', () => {
     const result = disposableEmailBlocklist();
 
-    expect(result).not.toEqual([]);
+    expect(Array.isArray(result)).toBe(true);
+    expect(result.length).toBeGreaterThan(0);
+    expect(result.every(value => typeof value === 'string')).toBe(true);
```

Comment:
Prettier has failed, can you just fix this part?

```suggestion
    expect(result.every((value) => typeof value === 'string')).toBe(true);
```

This command will automatically correct the problem.

```sh
> npm run lint:fix
```

Reply:
I've added a new commit for this! 👍

Reply:
Thank you!